.github/workflows/deploy-website.yaml

```
@@ -31,7 +31,7 @@ jobs:
           java-version: 11
 
       - name: Build Detekt Documentation
-        run: ./gradlew :detekt-generator:generateDocumentation --build-cache --stacktrace
+        run: ./gradlew :detekt-generator:generateDocumentation --stacktrace
```

```suggestion
        run: ./gradlew :detekt-generator:generateDocumentation
```
Let's remove --stacktrace since we are publishing Gradle build scan #3724